packages/did/src/DidBatcher/FullDidCreationBuilder.ts

```diff
@@ -92,28 +93,30 @@ export class FullDidCreationBuilder extends FullDidBuilder {
   }
 
   /**
-   * Consume the builder and delegates to the closure the [[SubmittableExtrinsic]] containing the details of a DID creation with the provided details.
+   * Use the builder and delegates to the callback the [[SubmittableExtrinsic]] containing the details of a DID creation with the provided details.
    *
    * @param signer The [[KeystoreSigner]] to sign the DID operation. It must contain the expected DID authentication key.
    * @param submitter The KILT address of the user authorised to submit the creation operation.
-   * @param handler A closure to submit the extrinsic and return the created [[FullDidDetails]] instance.
+   * @param callback A callback to submit the extrinsic and return the created [[FullDidDetails]] instance.
    * @param atomic A boolean flag indicating whether the whole state must be reverted in case any operation in the batch fails.
    *
-   * @returns The [[FullDidDetails]] as returned by the provided closure.
+   * @returns The [[FullDidDetails]] as returned by the provided callback.
    */
   /* istanbul ignore next */
-  public async consumeWithHandler(
+  public async buildAndSubmit(
     signer: KeystoreSigner,
     submitter: IIdentity['address'],
-    handler: FullDidCreationHandler,
+    callback: FullDidCreationCallback,
```

Comment:
not sure about the terminology here, I think the js crowd may expect this to be a result callback that is called when the operation finishes?

Reply:
@arty-name what would you say?

Reply:
Yeah the change from handler/closure to callback was suggested by Tom to make it easier for the JS audience to digest. I have 0 preference, so we could go for whatever you guys think it's best.

Reply:
My gripe was with "closure" specifically, because this word has a very precise different meaning in Javascript. Callback is the most universal term, handler is also well-known. While callbacks were indeed often used to report the result of an async operation, I don’t think it would confuse anyone here, especially in combination with docs.

Reply:
ok let's stick with it then